doc/env_variables.md

```diff
@@ -17,4 +17,5 @@
 * `ACPPL_RT_MAX_CACHED_NODES`: Maximum number of nodes that the runtime buffers before flushing work.
 * `ACPP_SSCP_FAILED_IR_DUMP_DIRECTORY`: If non-empty, hipSYCL will dump the IR of code that fails SSCP JIT into this directory.
 * `ACPP_RT_GC_TRIGGER_BATCH_SIZE`: Number of nodes in flight that trigger a garbage collection job to be spawned
-* `ACPP_RT_OCL_NO_SHARED_CONTEXT`: If set to a non-zero value, instructs the OpenCL backend to not attempt to construct a shared context across devices within a platform. This can be necessary on OpenCL implementations that do not support this. Note that if shared contexts are unavailable, support for data transfers between devices might be limited as the devices can no longer directly talk to each other.
\ No newline at end of file
+* `ACPP_RT_OCL_NO_SHARED_CONTEXT`: If set to a non-zero value, instructs the OpenCL backend to not attempt to construct a shared context across devices within a platform. This can be necessary on OpenCL implementations that do not support this. Note that if shared contexts are unavailable, support for data transfers between devices might be limited as the devices can no longer directly talk to each other.
+* `ACPP_RT_OCL_SHOW_ALL_DEVICES`: If set to a non-zero value, instructs the OpenCL backend to expose all found devices, even if those might be incompatible with AdaptiveCpp or unable to execute kernels.
```

Comment:
Nit: setting it to `2` (or `42`) does not work:

```suggestion
* `ACPP_RT_OCL_SHOW_ALL_DEVICES`: If set to `1`, instructs the OpenCL backend to expose all found devices, even if those might be incompatible with AdaptiveCpp or unable to execute kernels.
```

Reply:
Right, this is because we use `istream` with `bool` which is only well-defined for `0` and `1`. But the new variable is not the only one affected since all share the same infrastructure; I have thus also changed the documentation for another boolean variable which had a similar description.